cmd/utils/flags.go

```diff
@@ -1726,6 +1731,37 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 		cfg.NetworkId = *cfg.Genesis.GetNetworkID()
 	}
 
+	if ctx.GlobalIsSet(EthProtocolsFlag.Name) {
+		protocolVersions := SplitAndTrim(ctx.GlobalString(EthProtocolsFlag.Name))
+		if len(protocolVersions) > 0 {
+			for _, versionString := range protocolVersions {
+				version, err := strconv.ParseUint(versionString, 10, 0)
+				if err != nil {
+					continue
+				}
+
+				isValid := false
+				for _, proto := range eth.DefaultProtocolVersions {
+					if proto == uint(version) {
+						isValid = true
+					}
+				}
+
+				if !isValid {
+					Fatalf("--%s must be comma separated list of %s", EthProtocolsFlag.Name, strings.Join(strings.Fields(fmt.Sprint(eth.DefaultProtocolVersions)), ","))
+				}
+				cfg.ProtocolVersions = append(cfg.ProtocolVersions, uint(version))
```

Comment:
Do we need to do any further sanitation here? For example, should `--eth.protocols=65,65,65` work?